pipeline/load.py

```diff
@@ -0,0 +1,126 @@
+'''Module that inserts the transformed data into an RDS'''
+import os
+import psycopg2
+from pandas import Timestamp
+from psycopg2.extensions import connection, cursor
+from psycopg2.extras import RealDictCursor
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
+def get_connection() -> connection:
+    """ Establishes a connection with database. """
+    return psycopg2.connect(f"""dbname={os.getenv("DB_NAME")}
+                            user={os.getenv("DB_USER")}
+                            password={os.getenv("DB_PASSWORD")}
+                            host={os.getenv("DB_HOST")}
+                            port={os.getenv("DB_PORT")}""")
+
+
+def get_cursor(connect: connection) -> cursor:
+    """ Create a cursor to send and receive data. """
+    return connect.cursor(cursor_factory=RealDictCursor)
+
+
+def get_foreign_key(db_cursor: psycopg2.extensions.cursor, table_name: str,
+                    column_name: str, value: str) -> int:
+    """ Gets foreign keys. """
+
+    db_cursor.execute(
+        f"SELECT * FROM {table_name} WHERE {column_name} = '{value}'")
+    result = db_cursor.fetchone()
+    if result:
+        first_key = next(iter(result))
+        return result[first_key]
+    raise ValueError('Invalid Data!')
+
+
+def insert_into_earthquake(db_conn: connection,
+                           db_cursor: cursor,
+                           earthquake_data: list[dict]) -> None:
+    """Inserts cleaned data into the earthquake table."""
+    for earthquake in earthquake_data:
+        alert_id = get_foreign_key(
+            db_cursor, 'alerts', 'alert_type', earthquake['alert'])
+        magnitude_id = get_foreign_key(
+            db_cursor, 'magnitude', 'magnitude_type', earthquake['magnitude_type'])
+        network_id = get_foreign_key(
+            db_cursor, 'networks', 'network_name', earthquake['network'])
+        type_id = get_foreign_key(
+            db_cursor, 'type', 'type_name', earthquake['type'])
+        query = """
+            INSERT INTO earthquakes (time, tsunami, felt_report_count, magnitude, 
+                          cdi, latitude, longitude, detail_url, alert_id, magnitude_id, network_id, type_id) VALUES
+                          (%s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s)"""
+        values = (
+            earthquake['at'],
+            bool(earthquake['tsunami']),
+            int(earthquake['felt']),
+            float(earthquake['magnitude']),
+            float(earthquake['cdi']),
+            float(earthquake['latitude']),
+            float(earthquake['longitude']),
+            earthquake['event_url'],
+            alert_id,
+            magnitude_id,
+            network_id,
+            type_id
+        )
+        db_cursor.execute(query, values)
+
```

Comment:
Maybe use .executemany(), so it is quicker. If we append the values tuple to a list, that way it will be a list of tuples, and then use that in .executemany()?

```suggestion
        values = (
            earthquake['at'],
            bool(earthquake['tsunami']),
            int(earthquake['felt']),
            float(earthquake['magnitude']),
            float(earthquake['cdi']),
            float(earthquake['latitude']),
            float(earthquake['longitude']),
            earthquake['event_url'],
            alert_id,
            magnitude_id,
            network_id,
            type_id
        )
values_list.append(values)

db_cursor.execute(query, values_list)
```